src/stages/stage3_events.c

```diff
@@ -809,9 +810,9 @@ static int wriggle_rocket_laserbullet(Projectile *p, int time) {
 				.sprite = "blast",
 				.pos = p->pos,
 				.color = c,
-				.rule = timeout,
+				.timeout = 35 - 5 * frand(),
```

Comment:
This looks evil

Reply:
Exactly is evil as the code it replaces, it's a direct adaptation. I've just moved the timeout from args to the new property, exactly the same as in many other places. This is within a `PARTICLE` call, though, so it ought to not desync anything.